lib/src/edwards25519/extensions.dart

```diff
@@ -0,0 +1,25 @@
+extension ListExtension on List<int> {
+  int bit(int offset) => (this[offset >> 3] >> (offset & 7)) & 1;
+
+  int constantTimeEqual(List<int> other) {
```

Comment:
How can it be "constant time equal" if it's iterating through the list?

Reply:
I didn't know how to call it. It was called that way in the original code. To me also confusing, certainly this is not just equality. Can you suggest a name?